### PR TITLE
[GEOS-7349] value of default geometry always as GeoJSON geometry...

### PR DESCRIPTION
**Pull request by awaterme**
[GEOS-7349] value of default geometry always as GeoJSON geometry, never within properies

New pull request for [GEOS-7349](https://osgeo-org.atlassian.net/browse/GEOS-7349), after discussion on GS mailing list.

PR removed loop for finding alternative geometry if default geometry is null.

Also fixes writing "null" within properties for default geometry.

Corrected minor typos.

### CODE REVIEW

File: src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
- * (c) 2001 - 2013 OpenPlans
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2016 OpenPlans
```
Comment (aaime) [inline src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java]: Nope, this one has to stay up to 2013, it was the previous copyright owner

File: src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java [hunk 2]
```diff
@@ -212,30 +202,27 @@ else if ( id_option.length() != 0){
...
+                                // If it's the default that you already
+                                // printed above, so you don't need it here.
+                                if (ad.equals(defaultGeomType)) {
+                                    // Do nothing, we wrote it above
+                                    // jsonWriter.value("geometry_name");
+                                } else if(value == null){
                                     jsonWriter.key(ad.getLocalName());
-                                    jsonWriter.value(value);
+                                    jsonWriter.value(null);
+                                }else {
```
Comment (aaime) [inline src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java]: Formatting

File: src/wfs/src/test/java/org/geoserver/wfs/json/GeoJSONTest.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
-/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
- * (c) 2001 - 2013 OpenPlans
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2016 OpenPlans
```
Comment (aaime) [inline src/wfs/src/test/java/org/geoserver/wfs/json/GeoJSONTest.java]: Same as the other file, should stay 2013

**Comment by aaime** [comment]
Requires minor fixes to copyright headers and formatting, besides that, looks good to me.

**Comment by awaterme** [comment]
Fixed copyrights headers and formatting

**Comment by aaime** [comment]
Manually merged to squash the two commits into one, and backported to 2.8.x and 2.7.x

**Pull request closed without merging**